src/components/FormItem.vue

```diff
@@ -124,7 +138,8 @@ export default {
             errors: [],
             validating: false,
             isTextArea: type === 'textarea',
-            showFormErrors: !!this.formErrors,
+            showFormErrors: (this.formErrors.length > 0),
```

Comment:
velmi zle co ak je formErrors undefined ? - padne mi to na `Cannot read property 'length' of undefined`